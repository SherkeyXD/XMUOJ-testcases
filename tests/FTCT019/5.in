50
12 15 15 9 5 16 9 7 16 7 11 7 8 20 13 19 4 16 13 7 17 5 13 15 16 5 6 3 1 4 16 10 20 18 7 9 3 13 6 6 20 11 1 16 9 16 8 9 13 11 
