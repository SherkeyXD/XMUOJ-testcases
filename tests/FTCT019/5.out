2963
14774
